pkg/pdapi/pdetcd.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdapi
+
+import (
+	"context"
+	"crypto/tls"
+	"time"
+
+	etcdclientv3 "github.com/coreos/etcd/clientv3"
+)
+
+type PDEtcdApi interface {
+	PutKey(key, value string) error
+
+	DeleteKey(key string) error
+}
+
+type pdEtcdClient struct {
+	timeout    time.Duration
+	etcdClient *etcdclientv3.Client
+}
+
+func NewPdEtcdClient(url string, timeout time.Duration, tlsConfig *tls.Config) (PDEtcdApi, error) {
+	etcdClient, err := etcdclientv3.New(etcdclientv3.Config{
+		Endpoints:   []string{url},
+		DialTimeout: timeout,
+		TLS:         tlsConfig,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return &pdEtcdClient{
+		etcdClient: etcdClient,
+		timeout:    timeout,
+	}, nil
+}
+
+func (pec *pdEtcdClient) PutKey(key, value string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), pec.timeout)
+	defer cancel()
+	_, err := pec.etcdClient.Put(ctx, key, value)
```

Comment:
```suggestion
	return pec.etcdClient.Put(ctx, key, value)
```

why not return the original result?

Reply:
I think we don't need to handle the etcd response.  we only need to ensure the `PutKey` is success by check whether error is nil.